### DISCUSSION TITLE
Digital vs Analog Inputs for Novice

### DISCUSSION THREAD

**Discussion by NRKorsakov** [shanteacontrols/OpenDeck#258] [General]
Hi!

I am very new to the microcontroller world, but am very excited about building some midi custom controllers with the OpenDeck Software Platform.

I bought a Raspberry Pico to mess-around and learn on before i try anything ambitious, and successfully flashed the OpenDeck software to it and it shows up in the online Opendeck Configurator in Chrome on my MacBook.

I'm trying to learn now how to connect pots and buttons, and about the differences between digital and analog pins and how they differ or can be used for the same things?

Also if it is possible to get "mux" chips to connect to the digital pins to get additional inputs?

Is there somewhere on the Opendeck Github site that goes more in-depth about all of this?

I've read the page for Opendeck on the Pico, but really don't understand how many, what type, and which pins can be used for what?

Thanks!

**Comment by paradajz** [comment]
Everything you have asked is already covered in the wiki. https://github.com/shanteacontrols/OpenDeck/wiki

> I'm trying to learn now how to connect pots and buttons

https://github.com/shanteacontrols/OpenDeck/wiki/Supported-components
https://github.com/shanteacontrols/OpenDeck/wiki/Raspberry-Pi-Pico#connections

> Also if it is possible to get "mux" chips to connect to the digital pins to get additional inputs?

https://github.com/shanteacontrols/OpenDeck/wiki/Creating-custom-board-variant#analog

**Reply by NRKorsakov** [reply]
Hi and thanks for the reply!

I will check those out.

Thanks again!
